tests/unit/general/conftest.py

```diff
@@ -26,9 +26,31 @@
 from tests.unit.flask.constants import FLASK_CONTAINER_NAME
 from tests.unit.go.constants import DEFAULT_LAYER as GO_DEFAULT_LAYER
 from tests.unit.go.constants import GO_CONTAINER_NAME
+from tests.unit.test_charm.src.charm import TestCharm
 
 PROJECT_ROOT = pathlib.Path(__file__).parent.parent.parent.parent
 
+TEST_CONTAINER_NAME = "app"
+TEST_DEFAULT_LAYER = {
+    "services": {
+        "app": {
+            "override": "replace",
+            "startup": "enabled",
+            "command": "test-command",
+            "user": "_daemon_",
+        }
+    }
+}
+
+
+@pytest.fixture(name="generic_harness")
+def generic_harness_fixture():
```

Comment:
We should not use framework-tied harnesses in testing generic functions such as relations testing - a generic charm/harness has been created for such reasons.